Add temperature topic

Ktakao opened the pull request:
#14 
I added temperature topic. I publish [the temperature messeage](http://docs.ros.org/melodic/api/sensor_msgs/html/msg/Temperature.html) on /temperature, because the [xsens imu ros package](https://github.com/ethz-asl/ethzasl_xsens_driver) was doing it.

7675t commented:
Hi, thanks for your contribution.
Could you please add a parameter like 'publish_temperature' to enable / disable  the publish ? It would be much easier to merge your commit.
Thanks.

Ktakao commented:
Thank you for review and advise.

> I feel ti is reasonable that the temperature should be private on /imu/temperature. And other packages (http://wiki.ros.org/myahrs_driver , https://github.com/mdrwiega/bosch_imu_driver, http://wiki.ros.org/orientus_driver) do so.

Indeed. I updated that temperature is private. And I updated it possible to switch the publish temperature.

7675t commented:
Thank you for your contribution. This fix #17 .
